README.md

```diff
@@ -27,18 +27,15 @@ Complex table:
     <tr>
         <td rowspan=2>category 1</td>
         <td>line 2</td>
-        <td><b>plop</b></td>
```

Comment:
That row contains important informations!

Reply:
Maybe not.

Reply:
plop is very important